solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -43,16 +77,65 @@
       display: flex;
       flex-direction: column;
       justify-content: center;
+      position: relative;
+
+      .liquidity__info-tile__tooltip {
+        width: 12px;
+        height: 12px;
+        position: absolute;
+        top: 0px;
+        right: 8px;
+        cursor: pointer;
```

Comment:
`cursor: pointer` is unnecessary here. Trigger component in the `Tooltip` component provides `cursor: pointer`.